options.c

```diff
@@ -296,6 +296,7 @@ options_remove(struct options_entry *o)
 	else
 		options_value_free(o, &o->value);
 	RB_REMOVE(options_tree, &oo->tree, o);
+	free((void *)o->name);
```

Comment:
You do not need, and should not include, the cast. The few places in the code which do this cast should remove the cast, IMHO. This cast hasn't been needed for more than two decades.

Reply:
These are const casts and are necessary.